recipes/libpsl/all/conanfile.py

```diff
@@ -76,7 +79,11 @@ def generate(self):
         env.generate()
         tc = MesonToolchain(self)
         tc.project_options["runtime"] = self._idna_option
-        tc.project_options["builtin"] = self._idna_option
+        if Version(self.version) >= "0.21.5":
+            tc.project_options["builtin"] = self.options.enable_builtin
```

Comment:
If this option is only valid from this version onward, it should be removed in the `config_options()` method so as to not influcence the pkgid of older versions :)

Reply:
Also, this PR that tried to target an older release https://github.com/conan-io/conan-center-index/pull/22029/files#diff-308e90de1d7646e4db40d401de640dd72698dbb96cb43871ac4fb7bbd8eb8566R77

has it still check for the old ._idna_option. I take that yours is more extensible, but could you confirm what the original upstream intention is? Thanks! :)

Reply:
No it should not be removed, older versions of the lib used to allow setting the 'builtin' option to the same values as the 'runtime' option (libicu, libidn, etc) but now they changed the meaning/possible values to just True/False.
I preserved compatibility with older versions
https://github.com/conan-io/conan-center-index/blob/41dc055e5210007408ff51c09d02a9e7c1b2ebb7/recipes/libpsl/all/conanfile.py#L86

Reply:
Sorry took me a while to understand your comment :), fixed in https://github.com/conan-io/conan-center-index/pull/23642/commits/1e108a372847eaad594eab031be84212da3d81ea